views/partials/dashboards/remove-enroll.hbs

```diff
@@ -27,9 +27,15 @@
         <input id="" class="js-enroll-submit remove-enroll-submit" type="submit" value="{{getRemoveString "remove-enroll-submit"}}"/>
       </form>
     </div>
+    <div class="flx jst-cntr">
+        <button
```

Comment:
The default type for a `button` element is `submit`.  I _think_ you're safe here because the button is outside of the form element, but to be sure this button doesn't submit the form (on enter keypress, perhaps) you could add `type="button"`.

I've always found having to add `type="button"` to a `<button>` annoyingly redundant, but usually do it whenever a form is present on the page.

Reply:
Ah, that's good to know - that's part of the reason I did this stuff in a model is because I didn't want to deal with a button inside a form.